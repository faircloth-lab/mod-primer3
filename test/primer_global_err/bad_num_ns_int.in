SEQUENCE_ID=bad_num_ns_int
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_INTERNAL_NUM_NS_ACCEPTED=-1
=
