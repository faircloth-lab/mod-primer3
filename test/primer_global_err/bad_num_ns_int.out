SEQUENCE_ID=bad_num_ns_int
SEQUENCE_TEMPLATE=TAGCTGATGCTAGCTAGCGTGGGCATGCTAGCTAGCTGATGCAGCTAGCTGATCGTAGCTAGCTATCGATCGTAGCTGACTGATCGTAGCTAGCTGATCGATGCTAGCTAGCTAGCTGATCATCTACGTAGCTGACTATCGATGCTACTACTATCGATCGTAGCTAGCTGATCGTAGCTAGCTAGCTGATGCTAC
PRIMER_INTERNAL_NUM_NS_ACCEPTED=-1
PRIMER_ERROR=Illegal value for PRIMER_INTERNAL_OLIGO_NUM_NS
=
